style.css

```diff
@@ -504,6 +411,125 @@
     }
   }
 
+  .popup_window {
+    position: fixed;
+    top: 15px;
+    bottom: 15px;
+    width: 89vw;
+    padding: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: #fff;
+    z-index: 225;
+  }
+
+  .popup_title_container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .popup_title_container > .popup_title {
+    font-family: 'Roboto', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .languages {
+    list-style: none;
+    display: flex;
+    gap: 12px;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .texts {
+    font-family: 'Poppins', sans-serif;
+    font-size: 13px;
+    font-weight: 600;
+    line-height: 16px;
+    letter-spacing: 0;
+    text-align: left;
+    padding: 0.6rem 0.75rem;
+    border: 0.06rem solid #979493;
+    list-style: none;
+  }
+
+  .popup_content {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+  }
+
+  .popup_content > img {
+    width: 100%;
+    height: auto;
+    margin: auto;
+  }
+
+  .popup_content > p {
+    font-family: 'Inter', serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+  }
+
+  .button_container {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] It is essential as developers that we closely follow the given design we are working with, this would help us become better developers and to stand out so kindly ensure that the spacing between your buttons, button elements and the width of the button matches the Figma design

### Your design
<img width="711" alt="Screenshot 2023-07-02 at 00 22 00" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/8d37dac1-2503-4fdf-8b08-625c47c1c7fc">

### Figma design

![Screenshot 2023-07-02 at 00 22 00 (2)](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/4a0e3428-d05b-42bf-8e3d-584ce4c103de)